### ISSUE TITLE
Consider switching Xdmf to use official library

### ISSUE THREAD

**Issue by Yurlungur** [parthenon-hpc-lab/parthenon#1033]
While messing with #1019 I found that the XDMF official library now no longer requires VTK as a dependency and can probably built with only HDF5 as a dependency. It may be worth trying to switch our XDMF implementation to use the library.

However, if OpenPMD is really the future for us, I don't know how much effort I want to put into this support, as the XDMF API is painful and poorly documented. And our current implementation "works" though it isn't feature-complete for all of our capabilities.

**Comment by pgrete** [comment]
Agreed. If OpenPMD/ADIOS2 does the job for us in practice (so after enough testing and usage), we may not need to spent additional effort on a second library (though not depending on vtk makes it much more attractive).
